rpc/client/rpc_test.go

```diff
@@ -6,7 +6,8 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-	"github.com/tendermint/merkleeyes/iavl"
+
+	"github.com/tendermint/merkleeyes/iavl" //TODO use tendermint/iavl ?
```

Comment:
yes please. will take a bit of fanagling

Reply:
#713